air-script/tests/bitwise/bitwise.air

```diff
@@ -22,26 +22,28 @@ integrity_constraints:
     enf k1 * (s' - s) = 0
 
     # Enforce that input is decomposed into valid bits
-    enf a0^2 - a0 = 0
-    enf a1^2 - a1 = 0
-    enf a2^2 - a2 = 0
-    enf a3^2 - a3 = 0
-    enf b0^2 - b0 = 0
-    enf b1^2 - b1 = 0
-    enf b2^2 - b2 = 0
-    enf b3^2 - b3 = 0
+    enf a_limb[0]^2 - a_limb[0] = 0
+    enf a_limb[1]^2 - a_limb[1] = 0
+    enf a_limb[2]^2 - a_limb[2] = 0
+    enf a_limb[3]^2 - a_limb[3] = 0
+    enf b_limb[0]^2 - b_limb[0] = 0
+    enf b_limb[1]^2 - b_limb[1] = 0
+    enf b_limb[2]^2 - b_limb[2] = 0
+    enf b_limb[3]^2 - b_limb[3] = 0
 
     # Enforce that the values in the column a in the first row should be the aggregation of the
```

Comment:
nit: in the first row of column a.
I think it should be the first row of the current 8 row cycle, but I think this is probably too verbose.